component/admin/sql/install/mysql/install.sql

```diff
@@ -9,19 +49,25 @@ CREATE TABLE IF NOT EXISTS `#__redcore_oauth_clients` (
   `scope` TEXT,
   `user_id` varchar(80),
   PRIMARY KEY (`id`),
-  KEY `idx_client_id` (`client_id`)
-) DEFAULT CHARSET = utf8;
+  UNIQUE KEY `idx_client_id` (`client_id`)
```

Comment:
I added the one change for the table in the upgrade.sql but I left this second one change out... I will add it now